presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/EnvironmentOptions.java

```diff
@@ -32,6 +36,9 @@
     @Option(name = "--without-presto", title = "without Presto", description = "do not start presto-master")
     public boolean withoutPrestoMaster;
 
+    @Option(name = "--bind", description = "bind ports on localhost")
+    public boolean bindPorts = toBoolean(firstNonNull(System.getenv("PTL_BIND_PORTS"), "true"));
```

Comment:
make it false on default. Also I would remove PTL_BIND_PORTS env variable, I don't see a need it for right now.

Reply:
replied https://github.com/prestosql/presto/pull/3354#discussion_r404012260